Proposals/xgov-188.md

```diff
@@ -0,0 +1,152 @@
+---
+id: 188
```

Comment:
ID must be 192 to match PR https://github.com/algorandfoundation/xGov/pull/192/files